assets/app/views/browse/build.html

```diff
@@ -6,84 +6,26 @@
-                  <span ng-switch-when="Failed">{{build.status.startTimestamp | duration : build.status.completionTimestamp}}</span>
-                  <span ng-switch-when="Running">running for <duration-until-now timestamp="build.status.startTimestamp"></duration-until-now></span>
-                  <span ng-switch-when="New">waiting for <duration-until-now timestamp="build.metadata.creationTimestamp"></duration-until-now></span>
-                  <span ng-switch-when="Pending">waiting for <duration-until-now timestamp="build.metadata.creationTimestamp"></duration-until-now></span>
-                  <span ng-switch-default>
-                    <span ng-if="build.status.startTimestamp">{{build.status.startTimestamp | duration : build.status.completionTimestamp}}</span>
-                    <span ng-if="!build.status.startTimestamp">waited for {{build.metadata.creationTimestamp | duration : build.status.completionTimestamp}}</span>
-                  </span>
-                </span>
-              </dd>
-              <dt ng-if="buildConfigName">Build configuration:</dt>
-              <dd ng-if="buildConfigName"><a href="project/{{build.metadata.namespace}}/browse/builds/{{buildConfigName}}">{{buildConfigName}}</a></dd>
-              <dt>Build strategy:</dt>
-              <dd>{{build.spec.strategy.type}}</dd>
-              <dt>Builder image:</dt>
-              <dd>
-                <div class="build-detail" ng-switch="build.spec.strategy.type">
-                  <div ng-switch-when="Source" ng-if="build.spec.strategy.sourceStrategy.from">
-                    <span class="build-detail-label">Builder image:</span>{{build.spec.strategy.sourceStrategy.from | imageObjectRef : build.metadata.namespace}}
-                  </div>
-                  <div ng-switch-when="Docker">
-                    <div ng-if="build.spec.strategy.dockerStrategy.from">
-                      <span class="build-detail-label">Builder image:</span>{{build.spec.strategy.dockerStrategy.from | imageObjectRef : build.metadata.namespace}}
-                    </div>
-                  </div>
-                  <div ng-switch-when="Custom" ng-if="build.spec.strategy.customStrategy.from">
-                    <span class="build-detail-label">Builder image:</span>{{build.spec.strategy.customStrategy.from | imageObjectRef : build.metadata.namespace}}
-                  </div>
+              <tab active="selectedTab.logs">
+                <tab-heading>Logs</tab-heading>
+                <log-viewer
+                  logs="logs"
+                  loading="logsLoading"
+                  links="true"
+                  download="canShowDownload"></log-viewer>
+                <div ng-show="!logsLoading" class="text-muted">
+                  connection closed.
+                  <a ng-click="initLogs()">reconnect</a>
```

Comment:
ok, i can keep it but put the boolean switch in the error handler of the new streamer.